Jenkinsfile

```diff
@@ -132,6 +132,7 @@ ansiColor('xterm') {
             sh '''#!/bin/bash -ex
             source ~/.nvm/nvm.sh
             nvm use v8.9.1
+            git diff
```

Comment:
are you planning to keep this once you've get the jenkins issue worked out?

Reply:
I was planning on it. It'll give us some extra info if there's another of the same issue.